### DISCUSSION TITLE
请问zig是否是有国际政治倾向的编程语言，或者说它是不是由政治家编出来的语言

### DISCUSSION THREAD

**Discussion by hakur** [zigcc/forum#38] [新手提问] [answered]
如题，学了一个rust发现这玩意喜欢搞政治，像吃了苍蝇一样恶心。

**Comment by defp** [comment]
啥叫国际政治倾向

**Comment by jiacai2050** [comment, marked as answer]
不是 rust 的问题，是 rust foundation 的锅，rust core team 的人也吐槽过

**Reply by hakur** [reply]
原来如此

**Comment by hhstore** [comment]
- ✅ 一个成熟的`人/组织`, 有政治倾向和立场. 很正常.  
    - 没有, 才不正常. 先进的`组织`都有立场. 成语: `党同伐异`.
    - 过去, 只是不主动`暴露和表达 倾向/立场`. 
- ✅ 无法包容`和自己认知不同的政治倾向和立场`, 可能是你的问题, 不是别人的问题. 
    - 可以鸵鸟, 装没看见.(大多数人都是这样干的...
    - 可以去反驳. (表达你的认知...
    - 可以不用. (远离让你不适的`政治倾向`...
- ✅ 但, 试图以`开源技术与国际政治无关` 让别人不表达, 是徒劳
    - 让别人不表达, 本身就是`政治`, 剥夺他人讲话权利.
    - `政治幼稚`: 不存在和政治无关的`形态/生态/产品/组织`.
    - `言论自由`: 你管不着别人表达你不想听的内容.
- ✅  很多 Top 项目, 都不主动`涉政`.  ≠ 所有项目都`不允许涉政`.
    - 这是别人的选择. 
    - 不涉政, 有不涉政的利益(大厂项目, 为了钱/商业利益). 并不是`高尚`.
    - 谁主导, 谁说了算. 
    - 你主导的项目, 也可以表达你的立场.
- ✅  未来, 你可能会看到更多的项目/组织, `主动/被动`涉及政治. 
    - 明确表态立场(选边站队).
    - 模糊化立场, 本身就是讨好所有人 => 搞钱 => 更好的搞钱.

**Reply by hakur** [reply]
你贴这段，有一种试图说服我认可开源项目政治化的目的，不过目前理智的人还是多，不会再项目里面搞政治，最多蓝鸟上发两句支持。
政治立场会让我感到无比愤怒（想弄死对方），因为我的政治立场完全不是西方白左的立场。

**Reply by hakur** [reply]
哎，怎么感觉我们要吵起来了，你认可开源政治化，我不认可

**Reply by hhstore** [reply]
> 哎，怎么感觉我们要吵起来了，你认可开源政治化，我不认可

- 并不是我认可`开源政治化`.
- 是接受`未来会越来越政治化`的客观事实.
- 无需过多讨论, 过几年, 你就明白了.

**Comment by aprosail** [comment]
至少[Andrew Kelley](https://github.com/andrewrk)对待非盈利组织的态度还是挺正的。zig去中心化的设计思路也有助于避免政治化的影响。
